### DISCUSSION TITLE
Alternative to Cache tagging (clearing multiple cache keys)

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#52970] [Q&A]
Hi,

I'm trying to upgrade my application from Laravel 10 to Laravel 11. I'm aware that Laravel removed the docs in 10.x about cache tagging, but I'm trying to figure out how to manage caching then in a modern Laravel project without tagging.

For instance, when querying a model that's cached, how do we bust the cache for that model when it changes?

Most of my keys that are stored in cache also take into account some data about the request, so it would be great to somehow clear groups of cache keys?

What's the approach?

**Comment by henzeb** [comment]
I have a package for that, that might help you out: https://packagist.org/packages/henzeb/laravel-cache-index

**Reply by sts-ryan-holton** [reply]
There's no native way of doing this?
